### PR TITLE
Add article on routing.

### PR DESCRIPTION
**Pull request by rictic**
Once this gets the LGTM we'll publish carbon-route.

(important note, we should publish carbon-route before this is merged so that bower install is all good.)

### CODE REVIEW

File: 1.0/articles/routing.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+Note also, how little actual routing there is here. Almost all of the work is being done by other elements that are purpose built for their roles in the UI, and they're just data bound to &lt;carbon-route>. This is right and proper.
+
+You can see the core of the idea here, but we’ve jumped ahead a bit. I mean, where does the route in `route="{{route}}"` come from? What’s that `pattern` property doing, and how is it related to `data`?
+
+{%endraw%}
+
+## &lt;carbon-route>
+
+Let’s walk back a bit and consider the consider the routing problem, one piece at a time, beginning with just a &lt;carbon-route>. What does it do, and how does it work?
```
Comment (motss) [inline 1.0/articles/routing.md]: `Let's walk back a bit and consider the consider the` should be changed to 
`Let's walk back a bit and consider the`

Comment (rictic) [inline 1.0/articles/routing.md]: Good catch, thanks! Fixed

**Comment by rictic** [comment]
Converted element references from &lt;element-name> to `<element-name>` for consistency with the rest of the site.

**Comment by rictic** [comment]
Got the LGTM from Arthur over chat!

**Comment by arthurevans** [comment]
Here's one over github: LGTM!
:ship: :it:

**Pull request merged**